.editorconfig

```diff
@@ -16,6 +16,8 @@ indent_size = 2
 # Doc: https://youtrack.jetbrains.com/issue/IDEA-170643#focus=streamItem-27-3708697.0-0
 ij_java_imports_layout = java.**,|,javax.**,|,org.**,|,com.**,|,com.diffplug.**,|,*
 ij_java_use_single_class_imports = true
+ij_java_class_count_to_use_import_on_demand = 999
+ij_java_names_count_to_use_import_on_demand = 999
```

Comment:
IntelliJ didn't respect `ij_java_use_single_class_imports` without the counts for me. Feel free to remove this if you don't like it.